test/Microsoft.AspNet.Server.KestrelTests/HttpsConnectionFilterTests.cs

```diff
@@ -311,5 +310,17 @@ public class HttpsConnectionFilterTests
 #endif
             }
         }
+
+        private static string GetTestCertificatePath()
+        {
+            if (Directory.GetCurrentDirectory().Contains(".testPublish"))
```

Comment:
This is a dangerous assumptions. Maybe in the future we'll rename this folder. A suggestion for a better approach. Look for a project.json file in the current folder. If you don't find it. look in the parent folders until you find one. Then use that as the folder for the pfx file